src/core/xds/grpc/xds_server_grpc.cc

```diff
@@ -150,35 +150,41 @@ void GrpcXdsServer::JsonPostLoad(const Json& json, const JsonArgs& args,
       }
     }
   }
-
   // Parse "server_uri".
-  auto server_uri = LoadJsonObjectField<std::string>(json.object(), args,
-                                                     "server_uri", errors);
-  if (server_uri.has_value()) {
-    server_target_ = std::make_shared<GrpcXdsServerTarget>(
-        server_uri.value(), channel_creds_config);
+  std::string server_uri_target;
```

Comment:
I think this can just be:

```
std::string server_uri =
    LoadJsonObjectField<std::string>(json.object(), args, "server_uri", errors)
    .value_or("");
```